eclipse-collections-forkjoin/pom.xml

```diff
@@ -54,8 +54,21 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+	<!-- 
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.slf4j</groupId>
```

Comment:
See above question.

Reply:
This is reverted in the latest commit as well.